.github/workflows/automationTemplate.yaml

```diff
@@ -53,77 +53,85 @@ jobs:
+      - if: ${{ env.CANCEL_WORKFLOW == false }}
+        name: Trigger Dev SDK Pipeline 
         run: |
           echo  "${{ secrets.AZURE_DEVOPS_TOKEN }}" | az devops login --organization https://devdiv.visualstudio.com/
           az pipelines run --organization https://devdiv.visualstudio.com/ --project DevDiv --name Oryx-PlatformBinary-${{ env.PLATFORM_NAME_CAMEL_CASE }} --branch automation/add-new-${{ inputs.platformName }}-versions-${{ env.RELEASE_DATE }} > /tmp/sdkPipelineTriggerResponse.json
 
-      - name: Check Dev SDK Pipeline Ran Successfully
+      - if: ${{ env.CANCEL_WORKFLOW == false }}
+        name: Check Dev SDK Pipeline Ran Successfully
         run: |
           pipelineInvocationId=$( cat /tmp/sdkPipelineTriggerResponse.json | jq ".id" )
           ${{ github.workspace }}/build/tools/Automation/checkPipelineStatus.sh $pipelineInvocationId
 
-      - name: Trigger Runtime Pipeline 
+      - if: ${{ env.CANCEL_WORKFLOW == false }}
+        name: Trigger Runtime Pipeline 
         run: |
           echo  "${{ secrets.AZURE_DEVOPS_TOKEN }}" | az devops login --organization https://devdiv.visualstudio.com/
           az pipelines run --organization https://devdiv.visualstudio.com/ --project DevDiv --name Oryx-BaseImages-${{ env.PLATFORM_NAME_CAMEL_CASE }} --branch automation/add-new-${{ inputs.platformName }}-release-${{ env.RELEASE_DATE }} > /tmp/runtimePipelineTriggerResponse.json
 
-      - name: Check Runtime Pipeline Ran Successfully
+      - if: ${{ env.CANCEL_WORKFLOW == false }}
+        name: Check Runtime Pipeline Ran Successfully
         run: |
           pipelineInvocationId=$( cat /tmp/runtimePipelineTriggerResponse.json | jq ".id" )
           ${{ github.workspace }}/build/tools/Automation/checkPipelineStatus.sh $pipelineInvocationId
 
-      - name: Install yq
+      - if: ${{ env.CANCEL_WORKFLOW == false }}
+        name: Install yq
         run: sudo wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq
 
-      - name: Update Runtime Build Number
+      - if: ${{ env.CANCEL_WORKFLOW == false }}
+        name: Update Runtime Build Number
         run: |
-          yq --version
+          set -x
           buildNumber=$(cat /tmp/status.json | jq ".buildNumber" | tr -d '"')
-          echo "buildNumber: $buildNumber"
           platform=${{ inputs.platformName }}
           [[ "$platform" == "dotnet" ]] && platform="dot-net-core" || platform=$platform
-          echo "platform: $platform"
           yq -i "(.[] | select(.name==\"$platform-run-time-versions\") | .constants.$platform-runtime-base-tag) = $buildNumber" build/constants.yaml
+          
+          # run automation again to preserve constants.yaml YamlDotNet indentation
+          # since YamlDoNet and yq do not share indentation spacing
+          dotnet run --project build/tools/Automation/Automation.csproj ${RELEASE_DATE} ${{ github.workspace }}
```

Comment:
This will prevent spacing changes PRs for example: https://github.com/microsoft/Oryx/pull/1787/files

Reply:
I see that yq has an --indent parameter to control the number of spaces used for indentation. Would this be an option here?
https://mikefarah.gitbook.io/yq/usage/output-format

Reply:
I've tried `--indent 0` however the minimum seems to be `--indent 2` which is the default. Unfortunately, YamlDotNet does not have indentation support. So this would be our simplest workaround.

Reply:
OK makes sense. Thanks for considering this!